cypress/integration/common/search.js

```diff
@@ -0,0 +1,11 @@
+import { When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+When('I search for a specific word', () => {
```

Comment:
```suggestion
When('I search for {string}', (query) => {
```
No problem to have non-reusable step definitions, but in this case it would make sense to pass in the search query.

Reply:
We talked about that, and the difference between imperative and declarative steps, and decided to go with declarative for the time being, but it can always be changed, if you prefer imperative, but reusable steps. http://itsadeliverything.com/declarative-vs-imperative-gherkin-scenarios-for-cucumber